docs/semgrep-assistant/overview.md

```diff
@@ -0,0 +1,116 @@
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Semgrep Assistant overview
+
+Semgrep Assistant provides GPT-4-powered security recommendations to help you review, triage, and remediate your Semgrep findings.
+
+![Screenshot of Semgrep Assistant message in GitHub](/img/semgrep-assistant-github.png)
+*Figure*. Semgrep Assistant detects a false positive.
+
+## Support and availability
+
+Semgrep Assistant supports findings generated by Semgrep Code, Semgrep Supply Chain, and Semgrep Secrets, and, in many cases, requires using Semgrep Cloud Platform. It is not available on the Semgrep CLI.
+
+Semgrep Assistant is available to users of the following source code managers (SCMs):
+  * GitHub Cloud
+  * GitLab, including SaaS and self-managed plans
+
+Semgrep Assistant does not support the use of GitHub Enterprise Server (self-hosted).
+
+### Language support
+
+Semgrep Assistant supports [the same languages as Semgrep Code](/supported-languages).
+
+## Features
+
+### Component tags
+
+**Component tags** use GPT-4 to categorize a finding based on its function, such as:
+
+- Payments
+- User authentication
+- Infrastructure
+
+By categorizing your code through component tags, Semgrep Assistant can help you prioritize **high-risk issues**, such as remediating a code finding related to payments or user authentication.
+
+![Semgrep Assistant Component tag list](/img/assistant-component-tags.png)
+
+### Rules recommendations
+
+Semgrep Assistant suggests rule categories for your rules through the Assistant recommendations in the **Dashboard** page. To accept the suggestion, click the **Accept** button, and Semgrep Assistant automatically updates the rule with the suggested category.
+
+### Auto-triage
+
+Semgrep Assistant uses GPT-4's understanding of programming languages and libraries and your code and triage history to auto-triage findings and indicate if a finding can safely be ignored. For every recommendation to ignore a finding, Semgrep also provides guidance with an explanation on why this is the case.
+
+It can then surface its recommendations through PR or MR comments, so developers can fix or triage an issue without switching contexts. Semgrep Assistant can also send notifications through Slack.
+
+![Screenshot of Semgrep Assistant in a Slack notification](/img/semgrep-assistant-slack.png#md-width)
+    *Figure*. Semgrep Assistant in a Slack notification.
+
+You can view findings that Assistant suggests should be **Ignored** in Semgrep Cloud Platform's **Findings page**. <!-- #10438 -->
+
+### Autofix
+
+Semgrep Assistant can suggest [autofix](/writing-rules/autofix/) code snippets for Semgrep rules that don't have human-written autofix suggestions when it identifies a true positive. Assistant customizes the code snippets it provides based on previous feedback, if any, and your rule customizations. For example, if you have a custom rule that recommends a specific sanitizer, then Assistant will recommend its use in the autofix suggestion.
```

Comment:
On first read I was wondering if we were autofixing the rule (i.e. a badly written rule) or the code.